test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
```

Comment:
StatPlots is used in [imgcomp.jl](https://github.com/JuliaPlots/Plots.jl/blob/master/test/imgcomp.jl#L17) which is included by runtests and installed in [travis_commands.jl](https://github.com/JuliaPlots/Plots.jl/blob/master/test/travis_commands.jl#L11) anyway, though.